### PR TITLE
fix(tools): remove gitpod checks from the PR

### PR DESCRIPTION
**Pull request by raisedadead**

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -68,13 +68,13 @@ github:
...
     # add a check to pull requests (defaults to true)
-    addCheck: true
+    addCheck: false
     # add a "Review in Gitpod" button as a comment to pull requests (defaults to false)
     addComment: true
     # add a "Review in Gitpod" button to the pull request's description (defaults to false)
     addBadge: false
     # add a label once the prebuild is ready to pull requests (defaults to false)
-    addLabel: false
+    addLabel: true
```
Comment (raisedadead) [inline .gitpod.yml:77]:
```suggestion
    addLabel: ready-on-gitpod
```

**Comment by raisedadead** [comment]
GitPod has been stalling on sending the checks, and out housekeeping workflows remain unmerged while the check is pending. I do not think we need the check. Only the button as a comment (the image seems to be broken for it, all the time) should be enough to launch it. 

Additionally this change should add a PR if and when a pre-build is ready.

**Review by naomi-lgbt — APPROVED** [review]
Looks good to me. Broken image has been driving me bonkers, BTW 🤣

**Comment by raisedadead** [comment]
🤷🏽‍♂️ - I do not know what the deal with it is.

**Pull request merged**